src/lldp_syncd/__main__.py

```diff
@@ -1,15 +1,15 @@
 import logging.handlers
 import sys
 
-import swsssdk.util
+import sonic_py_common.util
```

Comment:
Please help do a force merge because code change in__main__.py not covered by UT.